### ISSUE TITLE
Crash on recent forge versions

### ISSUE THREAD

**Issue by brisingraerowing** [cyanobacterium/AdditionalLootTables#3]
``` text
---- Minecraft Crash Report ----

WARNING: coremods are present:
  LoadingPlugin (Quark-beta-18.jar)
  NowWithRendering (redstonepaste-mc1.9-1.7.3.jar)
  LoadingPlugin (RandomThings-MC1.9-3.7.jar)
  Reborn Core ASM (RebornCore-1.9-2.1.0.53-universal.jar)
  CXLibraryCore (cxlibrary-1.0.0.jar)
  MalisisCorePlugin (malisiscore-1.9-3.0.6.jar)
  MalisisSwitchesPlugin (malisisswitches-1.9-2.0.0.jar)
  ItemPatchingLoader (ItemPhysic Full 1.3 mc1.9.jar)
Contact their authors BEFORE contacting forge

// I bet Cylons wouldn't have this problem.

Time: 5/5/16 11:44 PM
Description: Initializing game

java.lang.NullPointerException: Initializing game
    at sun.reflect.UnsafeFieldAccessorImpl.ensureObj(Unknown Source)
    at sun.reflect.UnsafeObjectFieldAccessorImpl.get(Unknown Source)
    at java.lang.reflect.Field.get(Unknown Source)
    at cyano.alt.AdditionalLootTables.getPoolsFromLootTable(AdditionalLootTables.java:148)
    at cyano.alt.AdditionalLootTables.lambda$postInit$6(AdditionalLootTables.java:127)
    at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(Unknown Source)
    at java.util.stream.ReferencePipeline$2$1.accept(Unknown Source)
    at java.util.Iterator.forEachRemaining(Unknown Source)
    at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Unknown Source)
    at java.util.stream.AbstractPipeline.copyInto(Unknown Source)
    at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
    at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(Unknown Source)
    at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(Unknown Source)
    at java.util.stream.AbstractPipeline.evaluate(Unknown Source)
    at java.util.stream.ReferencePipeline.forEach(Unknown Source)
    at cyano.alt.AdditionalLootTables.postInit(AdditionalLootTables.java:108)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:561)
    at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
    at com.google.common.eventbus.EventBus.post(EventBus.java:275)
    at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:228)
    at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:206)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
    at com.google.common.eventbus.EventBus.post(EventBus.java:275)
    at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:135)
    at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:770)
    at net.minecraftforge.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:316)
    at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:513)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:346)
    at net.minecraft.client.main.Main.main(SourceFile:124)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
    at sun.reflect.UnsafeFieldAccessorImpl.ensureObj(Unknown Source)
    at sun.reflect.UnsafeObjectFieldAccessorImpl.get(Unknown Source)
    at java.lang.reflect.Field.get(Unknown Source)
    at cyano.alt.AdditionalLootTables.getPoolsFromLootTable(AdditionalLootTables.java:148)
    at cyano.alt.AdditionalLootTables.lambda$postInit$6(AdditionalLootTables.java:127)
    at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(Unknown Source)
    at java.util.stream.ReferencePipeline$2$1.accept(Unknown Source)
    at java.util.Iterator.forEachRemaining(Unknown Source)
    at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Unknown Source)
    at java.util.stream.AbstractPipeline.copyInto(Unknown Source)
    at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
    at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(Unknown Source)
    at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(Unknown Source)
    at java.util.stream.AbstractPipeline.evaluate(Unknown Source)
    at java.util.stream.ReferencePipeline.forEach(Unknown Source)
    at cyano.alt.AdditionalLootTables.postInit(AdditionalLootTables.java:108)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:561)
    at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
    at com.google.common.eventbus.EventBus.post(EventBus.java:275)
    at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:228)
    at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:206)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
    at com.google.common.eventbus.EventBus.post(EventBus.java:275)
    at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:135)
    at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:770)
    at net.minecraftforge.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:316)
    at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:513)

-- Initialization --
Details:
Stacktrace:
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:346)
    at net.minecraft.client.main.Main.main(SourceFile:124)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
    Minecraft Version: 1.9
    Operating System: Windows 10 (amd64) version 10.0
    Java Version: 1.8.0_92, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 1926008144 bytes (1836 MB) / 4174749696 bytes (3981 MB) up to 17072521216 bytes (16281 MB)
    JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx16G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn1G
    IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
    FML: Minecraft 1.9 MCP 9.23 Powered by Forge 12.16.1.1891 124 mods loaded, 124 mods active
    States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
    UCHIJ   mcp{9.19} [Minecraft Coder Pack] (minecraft.jar) 
    UCHIJ   FML{8.0.99.99} [Forge Mod Loader] (forge-1.9-12.16.1.1891.jar) 
    UCHIJ   Forge{12.16.1.1891} [Minecraft Forge] (forge-1.9-12.16.1.1891.jar) 
    UCHIJ   itemphysic{1.3.0} [ItemPhysic] (minecraft.jar) 
    UCHIJ   actuallyadditions{1.9-r30} [Actually Additions] (ActuallyAdditions-1.9-r30.jar) 
    UCHIJ   additionalbanners{1.1.0.9} [Additional Banners] (AdditionalBanners-1.9-1.1.0.10.jar) 
    UCHIE   alt{1.02} [Additional Loot Tables] (AdditionalLootTables_1.9-1.02.jar) 
    UCHIJ   appleagain{1.1} [Apples Again] (ApplesAgain-1.9.0-1.1.jar) 
    UCHIJ   backlytra{1.9-0.0.1} [Backlytra] (backlytra-1.9-0.0.1.jar) 
    UCHIJ   basemetals{2.2.2} [Base Metals] (BaseMetals_1.9-2.2.2.jar) 
    UCHIJ   golems{4.03} [Extra Golems] (ExtraGolems[1.9]-4.03.jar) 
    UCHIJ   golems_metal{4.03.1} [Base Metal Golems] (BaseMetalGolems[1.9]-4.03.1.jar) 
    UCHIJ   Baubles{1.2.0.2} [Baubles] (Baubles-1.9-1.2.0.2.jar) 
    UCHIJ   baublesstuff{2.0.7} [Baubles Stuff] (baublesstuff-2.0.7.jar) 
    UCHIJ   bdlib{1.11.0.14} [BD Lib] (bdlib-1.11.0.14-mc1.9.jar) 
    UCHIJ   BetterAchievements{0.2.5.19} [Better Achievements] (BetterAchievements-1.9-0.2.5.19.jar) 
    UCHIJ   BiomesOPlenty{4.0.2.2018} [Biomes O' Plenty] (BiomesOPlenty-1.9-4.0.2.2018-universal.jar) 
    UCHIJ   JEI{3.3.3.197} [Just Enough Items] (jei_1.9-3.3.3.197.jar) 
    UCHIJ   blockdrops{1.0.0} [Block Drops] (Block Drops-1.9-1.0.0.jar) 
    UCHIJ   bookshelf{1.2.0.226} [Bookshelf] (Bookshelf-1.9-1.2.0.237.jar) 
    UCHIJ   bookshelfapi{3.0.4} [Bookshelf] (Bookshelf-API-1.9-3.0.4.jar) 
    UCHIJ   BuildersGuides{1.9-0.2.0-beta} [Builder's Guides] (buildersguides-1.9-0.2.0-beta.jar) 
    UCHIJ   buildingbrickscompatvanilla{1.9-1.4.8} [Building Bricks Vanilla Compat] (BuildingBricks-1.9-1.4.8.jar) 
    UCHIJ   buildingbricks{1.9-1.4.8} [Building Bricks] (BuildingBricks-1.9-1.4.8.jar) 
    UCHIJ   Chameleon{1.9-2.1.3} [Chameleon] (Chameleon-1.9-2.1.3.jar) 
    UCHIJ   chisel{@VERSION@} [Chisel] (Chisel-MC1.9-0.0.4.13.jar) 
    UCHIJ   mcmultipart{1.1.1} [MCMultiPart] (MCMultiPart-1.1.1-universal.jar) 
    UCHIJ   covers{0.3.0.11} [Simple Covers] (covers-0.3.0.11-mc1.9.jar) 
    UCHIJ   craftelytra{1.0} [Craftable Elytra] (CraftableElytra-1.0.jar) 
    UCHIJ   craftingcraft{3.0.3} [CraftingCraft] (craftingcraft-mc1.9-3.0.3.jar) 
    UCHIJ   craftingtweaks{4.0.10} [Crafting Tweaks] (craftingtweaks-mc1.9-4.0.10.jar) 
    UCHIJ   MineTweaker3{3.0.5} [MineTweaker 3] (CraftTweaker-Customized-1.9-3.0.5.jar) 
    UCHIJ   creativecore{1.4.0} [CreativeCore] (CreativeCore v1.4 mc1.9.jar) 
    UCHIJ   excore{1.5.3} [EXCore] (Excore-1.5.3-1.9.jar) 
    UCHIJ   creeperawareness{2.0} [Creeper Awareness] (CreeperAwareness-2.0-1.9.jar) 
    UCHIJ   cxlibrary{1.0.0} [CXLibrary] (cxlibrary-1.0.0.jar) 
    UCHIJ   wonderfulwands{2.0.0} [Wonderful Wands] (CyanosWonderfulWands_1.9.0-2.0.0.jar) 
    UCHIJ   cyclopscore{0.5.7} [Cyclops Core] (CyclopsCore-1.9-0.5.7.jar) 
    UCHIJ   Waila{1.7.0} [Waila] (Waila-1.7.0-B3_1.9.jar) 
    UCHIJ   DarkUtils{1.1.0.32} [Dark Utilities] (DarkUtilities-1.9-1.1.0.33.jar) 
    UCHIJ   decorationmegapack{1.9.4} [Decoration Mega Pack] (decorationmegapack-1.9.4.jar) 
    UCHIJ   eggs{1.0} [Eat the Eggs!] (Eat the Eggs-1.9-0.9 beta.jar) 
    UCHIJ   ea{v2.1.6b-mc1.9.0} [Ebon Arts] (EbonArts-v2.1.6b-mc1.9.0.jar) 
    UCHIJ   ElecCore{1.4.270} [ElecCore] (ElecCore-1.9-1.4.270.jar) 
    UCHIJ   poweradvantage{2.1.0} [Power Advantage] (PowerAdvantage_1.9.0-2.1.0.jar) 
    UCHIJ   electricadvantage{2.0.3} [Electric Advantage] (ElectricAdvantage_1.9-2.0.3.jar) 
    UCHIJ   eplus{4.1.0.86} [Enchanting Plus] (EnchantingPlus-1.9-4.1.0.97.jar) 
    UCHIJ   enchiridion{3.1.0} [Enchiridion] (Enchiridion-1.9-3.1.0.jar) 
    UCHIJ   endercompass{1.2.3} [Ender Compass] (EnderCompass-1.9-1.2.3.jar) 
    UCHIJ   shetiphiancore{3.2.1} [ShetiPhian-Core] (shetiphiancore-1.9.0-3.2.1.jar) 
    UCHIJ   endertanks{1.3.2} [EnderTanks] (endertanks-1.9.0-1.3.2.jar) 
    UCHIJ   enderutilities{0.5.3} [Ender Utilities] (enderutilities-1.9-0.5.3.jar) 
    UCHIJ   FFS{1.9-1.5.1.1.2} [Fancy Fluid Storage] (Fancy Fluid Storage-1.9-1.5.1.1.2.jar) 
    UCHIJ   frt{5.0.0.1} [Fire's Random Things] (FiresRandomThings-5.0.0.1.jar) 
    UCHIJ   frtvc{2.2} [Fire's Random Things Version Checker] (FiresRandomThings-5.0.0.1.jar) 
    UCHIJ   fe{1.2} [Food Expansion] (FoodExpansion1.2-mc1.9.jar) 
    UCHIJ   foxlib{@VERSION@} [FoxLib] (FoxLib-1.9-0.9.0.jar) 
    UCHIJ   lit{1.9-1.0.11} [Little Insignificant Things] (LittleInsignificantThings-1.9-1.0.11.jar) 
    UCHIJ   SilentLib{1.0.5} [Silent Lib] (SilentLib-1.9-1.0.5-11.jar) 
    UCHIJ   SilentGems{2.0.8} [Silent's Gems] (SilentsGems-1.9-2.0.8-alpha-25.jar) 
    UCHIJ   gemulation{1.9-2.0.4} [Gemulation] (gemulation-1.9-2.0.4.jar) 
    UCHIJ   pressure{1.3.1.13} [Pressure Pipes] (pressure-1.3.1.13-mc1.9.jar) 
    UCHIJ   advgenerators{0.9.20.8} [Advanced Generators] (generators-0.9.20.8-mc1.9.jar) 
    UCHIJ   guideapi{1.9-2.0.0-30} [Guide-API] (Guide-API-1.9-2.0.0-30.jar) 
    UCHIJ   hopperducts{1.4.7} [Hopper Ducts] (hopperducts-mc1.9-1.4.7.jar) 
    UCHIJ   LunatriusCore{1.1.2.34} [LunatriusCore] (LunatriusCore-1.9-1.1.2.34-universal.jar) 
    UCHIJ   InGameInfoXML{2.8.1.83} [InGame Info XML] (InGameInfoXML-1.9-2.8.1.83-universal.jar) 
    UCHIJ   integrateddynamics{0.3.7} [Integrated Dynamics] (IntegratedDynamics-1.9-0.3.7.jar) 
    UCHIJ   japta{1.4.1} [JAPTA] (JAPTA-1.9-1.4.1.jar) 
    UCHIJ   journeymap{1.9-5.2.0b1} [JourneyMap] (journeymap-1.9-5.2.0b1-unlimited.jar) 
    UCHIJ   jeresources{0.4.5.25} [Just Enough Resources] (JustEnoughResources-1.9-0.4.5.25.jar) 
    UCHIJ   malisiscore{1.9-3.0.6} [MalisisCore] (malisiscore-1.9-3.0.6.jar) 
    UCHIJ   malisisadvert{1.9-3.0.0} [Malisis Advert] (malisisadvert-1.9-3.0.0.jar) 
    UCHIJ   malisisblocks{1.9-3.0.1} [Malisis Blocks] (malisisblocks-1.9-3.0.1.jar) 
    UCHIJ   malisisdoors{1.9-4.0.1} [MalisisDoors] (malisisdoors-1.9-4.0.1.jar) 
    UCHIJ   malisisswitches{1.9-2.0.0} [Malisis Switches] (malisisswitches-1.9-2.0.0.jar) 
    UCHIJ   mantle{1.9-0.10.0.jenkins132} [Mantle] (Mantle-1.9-0.10.0.jar) 
    UCHIJ   mineralogy{3.0.1} [Mineralogy] (Mineralogy_1.9.0-3.0.1.jar) 
    UCHIJ   moarsigns{3.0.0} [MoarSigns] (MoarSigns-1.9-3.0.0.jar) 
    UCHIJ   modernmetals{0.9.2.3} [Modern Metals] (ModernMetals-0.9.2.3.jar) 
    UCHIJ   moneymod{2.0.0} [Money Mod] (MoneyMod 2.0.0.jar) 
    UCHIJ   moreanvils{5.0.0.2} [More Anvils] (MoreAnvils-5.0.0.2.jar) 
    UCHIJ   moreanvilsvc{2.1} [More Anvils Version Checker] (MoreAnvils-5.0.0.2.jar) 
    UCHIJ   NashornLib{1.9.0-1.8.77-1.2.1} [Nashorn Library] (NashornLib-1.9.0-1.8.77-1.2.1.5.jar) 
    UCHIJ   tconstruct{1.9-2.3.0.jenkins193} [Tinkers' Construct] (TConstruct-1.9-2.3.0.jar) 
    UCHIJ   neotech{3.0.6} [NeoTech] (NeoTech-1.9-3.0.6.jar) 
    UCHIJ   nethermetals{1.0.4} [Nether Metals] (NetherMetals-1.0.4.jar) 
    UCHIJ   netherportalfix{1.0} [Nether Portal Fix] (netherportalfix-mc1.9-3.0.6.jar) 
    UCHIJ   OAReborn{1.9-2.0.3.0} [OpenAutomation REBORN!] (OAReborn-1.9-2.0.3.0.jar) 
    UCHIJ   OALasers{1.0} [OALasers] (OAReborn-1.9-2.0.3.0.jar) 
    UCHIJ   samsocean{1.0.0} [Classic Oceans] (OceanFloor-1.9-1.0.1.jar) 
    UCHIJ   fodc{1.8.1} [Ore Dictionary Converter] (OreDictionaryConverter-1.8.1.jar) 
    UCHIJ   packingtape{0.5.1} [Packing Tape] (Packing Tape-0.5.1.jar) 
    UCHIJ   PlaceableDusts{1.0} [Placeable Dusts] (PlaceableDusts-1.0.jar) 
    UCHIJ   plantmegapack{1.9.8} [Plant Mega Pack] (plantmegapack-1.9.8.jar) 
    UCHIJ   progressiveautomation{1.6.42} [Progressive Automation] (ProgressiveAutomation-1.9-1.6.42.jar) 
    UCHIJ   quantumflux{2.0.6} [QuantumFlux] (quantumflux-1.9-2.0.6.jar) 
    UCHIJ   Quark{beta-18} [Quark] (Quark-beta-18.jar) 
    UCHIJ   randomthings{3.7} [Random Things] (RandomThings-MC1.9-3.7.jar) 
    UCHIJ   reap{1.0.4} [Reap Mod] (ReapMod 1.0.4.jar) 
    UCHIJ   reborncore{2.1.0.53} [RebornCore] (RebornCore-1.9-2.1.0.53-universal.jar) 
    UCHIJ   redstonepaste{1.7.3} [Redstone Paste] (redstonepaste-mc1.9-1.7.3.jar) 
    UCHIJ   rftools{4.46} [RFTools] (rftools-1.9.0-4.46.jar) 
    UCHIJ   rftoolsdim{4.23beta37} [RFTools Dimensions] (rftoolsdim-1.9.0-4.23beta37.jar) 
    UCHIJ   Roguelike{1.5.0} [Roguelike Dungeons] (RoguelikeDungeons-1.9-1.5.0.jar) 
    UCHIJ   routerreborn{1.9-4.0.1.11b_universal} [Router Reborn] (RouterReborn-1.9-4.0.1.11b_universal.jar) 
    UCHIJ   SaltMod{1.8.c} [Salty Mod] (SaltyMod-1.9-1.8.c_beta-1.jar) 
    UCHIJ   sciapi{1.3.0.3} [SciAPI] (SciAPI-1.3.0.3.jar) 
    UCHIJ   sensorcraft{0.2.1} [SensorCraft] (sensorcraft-0.2.1-MC1.9.jar) 
    UCHIJ   ShieldRework{1.2-MC1.9} [Shield Rework] (shieldrework-1.2.jar) 
    UCHIJ   redstonejukebox{4.0-BETA} [Redstone Jukebox] (SidbenRedstoneJukebox[1.9]-4.0-BETA.jar) 
    UCHIJ   steamadvantage{2.0.3} [Steam Advantage] (SteamAdvantage_1.9-2.0.3.jar) 
    UCHIJ   stellarsky{0.3.2.5b} [Stellar Sky] (Stellar Sky-0.3.2.5b.jar) 
    UCHIJ   StorageDrawers{1.9-3.1.6} [Storage Drawers] (StorageDrawers-1.9-3.1.6.jar) 
    UCHIJ   storagesilo{1.2.2} [StorageSilo] (storagesilo-1.9-1.2.2.jar) 
    UCHIJ   Tails{1.9-1.6.1} [Tails] (Tails-1.9-1.6.1.jar) 
    UCHIJ   techreborn{1.0.6.197} [TechReborn] (TechReborn-1.9-1.0.6.197-universal.jar) 
    UCHIJ   terrariabuttons{1.0.1} [Terraria Buttons] (TerrariaButtons-1.9-1.0.1.jar) 
    UCHIJ   TODOList{1.0.3} [TODO List] (TODOList-1.0.3.jar) 
    UCHIJ   Trashcans{2.0.2} [Trashcans] (Trashcans-1.9-2.0.2.jar) 
    UCHIJ   uncraftingtable{1.0.4} [Uncrafting Grinder] (UncraftingGrinder-1.9-1.0.4.jar) 
    UCHIJ   vending{1.3.1} [vending] (vending-1.9-1.3.4.jar) 
    UCHIJ   worldedit{6.1.1} [WorldEdit] (worldedit-forge-mc1.9-6.1.1-dist.jar) 
    UCHIJ   WorldEditCuiFe{1.0.2} [WorldEdit CUI Forge Edition] (WorldEditCuiFe-v1.0.2-mf-1.9-12.16.0.1865.jar) 
    Loaded coremods (and transformers): 
LoadingPlugin (Quark-beta-18.jar)
  vazkii.quark.base.asm.ClassTransformer
NowWithRendering (redstonepaste-mc1.9-1.7.3.jar)
  net.fybertech.nwr.NowWithRendering
LoadingPlugin (RandomThings-MC1.9-3.7.jar)
  lumien.randomthings.asm.ClassTransformer
Reborn Core ASM (RebornCore-1.9-2.1.0.53-universal.jar)
  reborncore.asm.RebornClassTransformer
CXLibraryCore (cxlibrary-1.0.0.jar)
  cubex2.cxlibrary.CoreModTransformer
MalisisCorePlugin (malisiscore-1.9-3.0.6.jar)
  net.malisis.core.util.chunkcollision.ChunkCollisionTransformer
  net.malisis.core.util.chunkblock.ChunkBlockTransformer
  net.malisis.core.renderer.transformer.MalisisRendererTransformer
  net.malisis.core.renderer.icon.asm.TextureMapTransformer
MalisisSwitchesPlugin (malisisswitches-1.9-2.0.0.jar)
  net.malisis.switches.asm.MalisisSwitchesTransformer
ItemPatchingLoader (ItemPhysic Full 1.3 mc1.9.jar)
  com.creativemd.itemphysic.ItemTransformer
    GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.5.0 NVIDIA 353.54' Renderer: 'GeForce GTX 980M/PCIe/SSE2'
    Pulsar/tconstruct loaded Pulses: 
        - TinkerCommons (Enabled/Forced)
        - TinkerWorld (Enabled/Not Forced)
        - TinkerTools (Enabled/Not Forced)
        - TinkerSmeltery (Enabled/Not Forced)
        - TinkerGadgets (Enabled/Not Forced)
        - TinkerOredict (Enabled/Forced)
        - TinkerIntegration (Enabled/Forced)
        - TinkerFluids (Enabled/Forced)
        - TinkerMaterials (Enabled/Forced)
        - craftingtweaksIntegration (Enabled/Not Forced)

    List of loaded APIs: 
        * actuallyadditionsapi (8) from ActuallyAdditions-1.9-r30.jar
        * Baubles|API (1.1.1.0) from Baubles-1.9-1.2.0.2.jar
        * BetterAchievements|API (0.2.5.19) from BetterAchievements-1.9-0.2.5.19.jar
        * ChiselAPI (0.0.1) from Chisel-MC1.9-0.0.4.13.jar
        * ChiselAPI|Carving (0.0.1) from Chisel-MC1.9-0.0.4.13.jar
        * CoFHAPI (1.8.9R1.2.0B1) from ActuallyAdditions-1.9-r30.jar
        * CoFHAPI|block (1.8.9R1.2.0B1) from ProgressiveAutomation-1.9-1.6.42.jar
        * CoFHAPI|energy (1.8.9R1.2.0B1) from RouterReborn-1.9-4.0.1.11b_universal.jar
        * CoFHAPI|tileentity (1.8.9R1.2.0B1) from ProgressiveAutomation-1.9-1.6.42.jar
        * commoncapabilities|api (0.0.1) from IntegratedDynamics-1.9-0.3.7.jar
        * CraftingTweaks|API (4.1) from craftingtweaks-mc1.9-4.0.10.jar
        * Creeper Awareness | API (2.0) from CreeperAwareness-2.0-1.9.jar
        * Creeper Awareness | API | EVENTS (2.0) from CreeperAwareness-2.0-1.9.jar
        * ElecCore|Abilities (#API_VER#) from ElecCore-1.9-1.4.270.jar
        * Guide-API|API (1.9-2.0.0-30) from Guide-API-1.9-2.0.0-30.jar
        * IC2API (1.0) from RebornCore-1.9-2.1.0.53-universal.jar
        * integrateddynamics|api (0.1.0) from IntegratedDynamics-1.9-0.3.7.jar
        * jeresources|API (0.4.5.25) from JustEnoughResources-1.9-0.4.5.25.jar
        * journeymap|client-api (1.2-SNAPSHOT) from journeymap-1.9-5.2.0b1-unlimited.jar
        * journeymap|client-api-display (1.2-SNAPSHOT) from journeymap-1.9-5.2.0b1-unlimited.jar
        * journeymap|client-api-event (1.2-SNAPSHOT) from journeymap-1.9-5.2.0b1-unlimited.jar
        * journeymap|client-api-model (1.2-SNAPSHOT) from journeymap-1.9-5.2.0b1-unlimited.jar
        * journeymap|client-api-util (1.2-SNAPSHOT) from journeymap-1.9-5.2.0b1-unlimited.jar
        * JustEnoughItemsAPI (3.1.0) from jei_1.9-3.3.3.197.jar
        * McJtyLib (1.9.0-1.8.3) from mcjtylib-1.9.0-1.8.3.jar
        * MoarSigns|API (1.1) from MoarSigns-1.9-3.0.0.jar
        * OAReborn|API (0.1) from OAReborn-1.9-2.0.3.0.jar
        * OAReborn|LasersAPI (0.1) from OAReborn-1.9-2.0.3.0.jar
        * pressureAPI (1.3.1.13) from pressure-1.3.1.13-mc1.9.jar
        * reborncoreAPI (2.1.0.53) from RebornCore-1.9-2.1.0.53-universal.jar
        * SensorCraftAPI (2) from sensorcraft-0.2.1-MC1.9.jar
        * stellarsky|API (2.2) from Stellar Sky-0.3.2.5b.jar
        * StorageDrawersAPI (1.7.10-1.2.0) from StorageDrawers-1.9-3.1.6.jar
        * StorageDrawersAPI|config (1.7.10-1.2.0) from StorageDrawers-1.9-3.1.6.jar
        * StorageDrawersAPI|event (1.7.10-1.2.0) from StorageDrawers-1.9-3.1.6.jar
        * StorageDrawersAPI|inventory (1.7.10-1.2.0) from StorageDrawers-1.9-3.1.6.jar
        * StorageDrawersAPI|pack (1.7.10-1.2.0) from StorageDrawers-1.9-3.1.6.jar
        * StorageDrawersAPI|registry (1.7.10-1.2.0) from StorageDrawers-1.9-3.1.6.jar
        * StorageDrawersAPI|render (1.7.10-1.2.0) from StorageDrawers-1.9-3.1.6.jar
        * StorageDrawersAPI|storage (1.7.10-1.2.0) from StorageDrawers-1.9-3.1.6.jar
        * StorageDrawersAPI|storage-attribute (1.7.10-1.2.0) from StorageDrawers-1.9-3.1.6.jar
        * TailsAPI (1.1.1) from Tails-1.9-1.6.1.jar
        * techrebornAPI (1.0.6.197) from TechReborn-1.9-1.0.6.197-universal.jar
        * WailaAPI (1.3) from Waila-1.7.0-B3_1.9.jar
    Launched Version: 1.9-forge1.9-12.16.1.1891
    XRay Version: 15
    LWJGL: 2.9.4
    OpenGL: GeForce GTX 980M/PCIe/SSE2 GL version 4.5.0 NVIDIA 353.54, NVIDIA Corporation
    GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

    Using VBOs: No
    Is Modded: Definitely; Client brand changed to 'fml,forge'
    Type: Client (map_client.txt)
    Resource Packs: 
    Current Language: English (US)
    Profiler Position: N/A (disabled)
    CPU: 8x Intel(R) Core(TM) i7-4710HQ CPU @ 2.50GHz
```

I have forge 12.16.1.1981 on MC 1.9

**Comment by DrPlantabyte** [comment]
I'm still on Forge build 1865, so I'll update and see if I can fix this.

EDIT: I updated to Forge-1.9-12.16.1.1894 (the latest available on the Forge website) and I'm still not getting the crash. Try updating again and see if that fixes it. If not, then it may be a mod-interaction issue.

**Comment by Syndaryl** [comment]
I was on 1865 and didn't have any errors. I updated to 1898, and I'm getting this crash now. :/

**Comment by AxolotlAccount** [comment]
it happens a crash like this, but NOT same! (forge-1.9-12.16.1.1900)

http://pastebin.com/nTiTaQEX

**Comment by Syndaryl** [comment]
Forge changed how loot works; this can't be used with Forge builds after 1885 until we get a new build.

**Comment by DrPlantabyte** [comment]
Fixed in version version 1.03 (and 1.04).

I had to use a little Voodoo to work around the new loot table event system, but it should still cooperate with other mods.
